TypeError: 'in <string>' requires string as left operand, not NoneType on Login

**Issue Summary**
Error on Login.

**Environment**
- OS: [Linux]
- API version: [7.4.4]
- Deployment: [Docker, self-hosted]
- For self-hosted, WSGI environment: [nginx/uwsgi]

- Database: [Postgres]

- Server config:
  Auth enabled? Yes
  Auth provider? Basic
  Customer views? No

- web UI version: [eg. 7.4.4]
- CLI version: [eg. 7.4.0]

**To Reproduce**
Steps to reproduce the behavior:
1. 
$export DATABASE_URL=postgres://db:5432/alerta
$ docker run --name alerta-web -e DATABASE_URL=$DATABASE_URL -e ADMIN_USERS=admin -e ADMIN_PASSWORD=password  --link postgres:latest \
-d -p 8080:8080 alerta/alerta-web:latest
2.  Try to login with admin user
3. See error (include command output)

```
2020-02-11 12:03:23,407 flask.app[38]: [ERROR] 'in <string>' requires string as left operand, not NoneType request_id=383e075e-f312-4e65-be83-c1165a68f7ea ip=xxx
Traceback (most recent call last):
  File "/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/venv/lib/python3.7/site-packages/flask_cors/decorator.py", line 128, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "/venv/lib/python3.7/site-packages/alerta/auth/basic.py", line 76, in login
    user = User.check_credentials(username, password)
  File "/venv/lib/python3.7/site-packages/alerta/models/user.py", line 183, in check_credentials
    user = User.find_by_username(username)
  File "/venv/lib/python3.7/site-packages/alerta/models/user.py", line 147, in find_by_username
    return User.from_db(db.get_user_by_username(username))
  File "/venv/lib/python3.7/site-packages/alerta/models/user.py", line 135, in from_db
    return cls.from_record(r)
  File "/venv/lib/python3.7/site-packages/alerta/models/user.py", line 127, in from_record
    email_verified=rec.email_verified
  File "/venv/lib/python3.7/site-packages/alerta/models/user.py", line 31, in __init__
    self.roles = ['admin'] if self.email in current_app.config['ADMIN_USERS'] else (roles or ['user'])
TypeError: 'in <string>' requires string as left operand, not NoneType
```

**Expected behavior**
Accepted Login

**Screenshots**

--- a/alerta/models/user.py
+++ b/alerta/models/user.py
@@ -29,7 +29,7 @@ class User:
         self.password = password  # NB: hashed password
         self.email = email
         self.status = kwargs.get('status', None) or 'active'  # 'active', 'inactive', 'unknown'
-        self.roles = [DEFAULT_ADMIN_ROLE] if self.email in current_app.config['ADMIN_USERS'] else (roles or ['user'])
+        self.roles = [DEFAULT_ADMIN_ROLE] if self.email and self.email in current_app.config['ADMIN_USERS'] else (roles or ['user'])
         self.attributes = kwargs.get('attributes', None) or dict()
         self.create_time = kwargs.get('create_time', None) or datetime.utcnow()
         self.last_login = kwargs.get('last_login', None)

